master/reference/public-cloud/digital-ocean.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Calico on Digital Ocean
+---
+
+Calico is designed to provide high performance massively scalable virtual
+networking for containers in public cloud environments such as Digital Ocean.
+
+## How to Run Calico in Digital Ocean
+
+Digital Ocean offers two networking options for droplets.
+Calico can be launched on either:
+
+- Public Interface
+- Private Interface [Optional]
+
+We recommend running Calico on the private interface, as it enhances security by
+ensuring Calico services are not accessible from the wider internet.
+
+However, hosts in different Datacenter Regions will not have IP connectivity with one
+another on their private address, and therefore will not be able to establish
```

Comment:
-> "another through their private address, and therefore they will not be able to establish"